internal/cmd/local/k8s/client.go

```diff
@@ -298,9 +301,14 @@ func (d *DefaultK8sClient) ServerVersionGet() (string, error) {
 }
 
 func (d *DefaultK8sClient) ServiceGet(ctx context.Context, namespace string, name string) (*corev1.Service, error) {
+	d.ClientSet.CoreV1().Services(namespace).List(ctx, metav1.ListOptions{})
```

Comment:
What's happening here? Is this result getting dropped on purpose?

Reply:
:doh: this was my quick poc to see how the `List` method worked that I didn't fully remove